### ISSUE TITLE
Is Midori developed on Github or Gitlab?

### ISSUE THREAD

**Issue by KaiserBarbarossa** [midori-browser/core#443]
Hi,

I have got a little question about Midori developing. It seems to me like there are two "Midoris". One [here](https://github.com/midori-browser/core) on GitHub and one [here](https://gitlab.com/midori-web/midori-desktop) on GitLab. Which one is the Midori that is in Ubuntu repositories for a long time?

This repo points to [https://www.midori-browser.org/](https://www.midori-browser.org/) as project website and that website is pointing to the gitlab version as repository. Is this repository here on Github abadoned and Midori moved to GitLab?

**Comment by mechanicjay** [comment]
It appears to me that these are two completely separate and unrelated projects.   The GitLab repo is built as a electron app wrapper.  This project is decidedly not.

Perhaps the original registrant of midori-browser.org lost control of the domain?

In which case, that should be fixed.

This does raise the larger issue, that the maintainer of this project seems to be somewhat inactive.

**Comment by mid-kid** [comment]
I stumbled across this on F-Droid, because I was looking for any new browser apps, and it's using the original logo on there. Mind you, it's a Lightning Browser fork without any mention of being one.

The weird part is that this takeover has gone without explanation, making it extremely sketchy in my eyes. Even the [wikipedia](https://en.wikipedia.org/wiki/Midori_(web_browser)) has been edited, but no trace of any real explanation as to why this happened. Not to mention, the website is full of things that don't mean anything, just look at it.

I would really love to know what's going on, because any way I look at this, I don't like it.

EDIT: I was linked [this article](https://astian.org/en/blog/2020/11/18/midori-web-browser-and-its-evolution/) on IRC, which raises a bunch of red flags in itself, but I guess it's something.

**Comment by mechanicjay** [comment]
Ugh.   That article makes it seem somewhat official?  But yeah, the no explanation thing, along with changing the entire architecture is what makes it sketch.

I need a light-weight webkit based browser -- I can't actually use an electron/chromium browser in my use case -- First, the tool chains just don't exist to built it, Second, the resource constrained environment wouldn't run it.

Guess it's time to fork this and try to rally devs around the new fork.

**Comment by mid-kid** [comment]
Yeah, I don't think Electron is the right technology despite the new project readme's justification. Electron is geared towards desktop applications and has a bunch of APIs to access system resources that a regular browser either wouldn't have, or would heavily restrict. I'm also not aware of methods to isolate websites within an electron app.

While I will miss Midori, I think there's enough browsers to fill its void and avoid taking up development. Have you considered Falkon (qtwebengine), Otter Browser (qtwebkit/qtwebengine), Qutebrowser (qtwebkit/qtwebengine), Epiphany (webkitgtk), Luakit (webkitgtk), surf (webkitgtk) or hell, even NetSurf (lightweight custom engine)?

**Comment by jamesmortensen** [comment]
From looking at the repositories, it appears both the WebKit version and the electron version of Midori have been abandoned.  However, check out [Wexond](https://github.com/wexond/browser-base).  This Midori repository is actually based off of the work done here.  Both Wexond and Midori appear to be identical in concept, except Wexond is actively developed.

**Comment by stellarpower** [comment]
Thanks for the recommendation, although, as of only a month ago [looks like it'll be abandonware](https://github.com/wexond/browser-base/issues/582#issuecomment-909378061)

**Comment by vandys** [comment]
I'd hate to see Midori desktop fade away. Is this github going to become the center of gravity for it having an ongoing life?

**Comment by jamesmortensen** [comment]
There is still Epiphany, based on WebKit. That's what the original Midori was based on too.

**Comment by Vistaus** [comment]
> There is still Epiphany, based on WebKit. That's what the original Midori was based on too.

I thought Midori was originally based on or spun off of Galeon?

**Comment by jamesmortensen** [comment]
I don't think so.  I googled it, and Galeon was a Gecko browser.  Midori was built on the WebkitGTK engine until this other group took over and decided to write it in Electron.  Maybe I'm wrong.  I'd never heard of Galeon before.
